build.gradle

```diff
@@ -309,20 +276,17 @@ task sourceJar(type: Jar) {
         from it.java
         from it.resources
     }
-	classifier = "sources"
+    archiveClassifier = "sources"
+    duplicatesStrategy = DuplicatesStrategy.INCLUDE
```

Comment:
Which resources are duplicate? Presumably it's not actually a source file but curious nevertheless

Reply:
> Entry META-INF/services/org.spongepowered.tools.obfuscation.service.IObfuscationService

Might actually want to merge that so it matches the fat jar.